src/Command.ino

```diff
@@ -198,6 +198,7 @@ String doExecuteCommand(const char *cmd, struct EventStruct *event, const char *
     }
     case 's': {
       COMMAND_CASE(       "save", Command_Settings_Save,   0); // Settings.h
+	    COMMAND_CASE(  "subscribe", Command_MQTT_Subscribe,  1);  // MQTT.h  //MFD: adding subscrie command
```

Comment:
should be with `#ifdef USES_MQTT`

Reply:
fixed.
Should I create a new pull request ?

Reply:
no you can edit this one, even by browser